<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>mcDialog</class>
 <widget class="QDialog" name="mcDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MATC</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >
    <normaloff>:/icons/Mesh3D.png</normaloff>:/icons/Mesh3D.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="mathLabel" >
       <property name="maximumSize" >
        <size>
         <width>45</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string>math:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mcEdit" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="mcHistory" />
   </item>
   <item>
    <widget class="QTextEdit" name="mcOutput" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>261</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mcOK" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-ok-apply" />
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
